### PR TITLE
Custom form login

### PR DESCRIPTION
**Pull request by ChiragJS**
Added Status Codes for failed login
@SundarakrishnanN check if it works fine for successful login too

### CODE REVIEW

File: .dockerignore
Comment (ChiragJS) [inline .dockerignore:1]: added this by mistake here, i was supposed to add it to .gitignore, just noticed rn,forgot to delete

**Review by SundarakrishnanN — APPROVED** [review]
LGTM

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (chrisvrose) [inline .dockerignore:1]: Delete file if not important

Comment (ChiragJS) [reply .dockerignore]: made the changes

**Pull request merged**